tests/tests.py

```diff
@@ -51,11 +52,11 @@ def test_synonym_valid_phrase(self):
     def test_synonym_not_valid_phrase(self):
         current_result = vb.synonym("sxsw")
         self.assertFalse(current_result)
-        
+
     def test_antonym_valid_phrase_1(self):
         current_result = vb.antonym("love")
         result = '{"text": ["hate"]}'
-        expected_result = json.loads(result)
+        expected_result = json.dumps(json.loads(result))
```

Comment:
This is so all responses are equivalent, every method would return a json object